igvc_platform/src/imu/YostLabDriver.cpp

```diff
@@ -168,16 +170,20 @@ std::string YostLabDriver::getMIMode()
     else
       return "Unknown";
   }();
-  ROS_INFO_STREAM(logger << "MI Mode: " << ret_buf << ", buf is: " << buf);
+  ROS_INFO_STREAM(this->logger << "MI Mode: " << ret_buf << ", buf is: " << buf);
```

Comment:
I'm only saying it because I'm scared of what Oswin would say if he saw `this` again, but I don't think it's necessary.